src/components/themeToggle/ThemeToggle.tsx

```diff
@@ -0,0 +1,26 @@
+import { useState, useEffect } from 'react';
+
+const ThemeToggle = () => {
+    const [theme, setTheme] = useState(localStorage.getItem('theme') || 'light');
```

Comment:
this is a good use case to create a custom hook (the one we reviewed in class `usePersistedState` should help